### ISSUE TITLE
Make Purchase Orders receipt look like other receipts

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#4303]
Our purchase orders have a non-standard presentation.  The reference/identifier is on the center of the page, rather than on the top right corner, like all other receipts.  Here is an example of a voucher.

![image](https://user-images.githubusercontent.com/896472/79265824-9ba59f80-7e8e-11ea-884e-f2d3355a5b59.png)

And here is the changes that need to be made to the Purchase Orders:
![image](https://user-images.githubusercontent.com/896472/79265677-64cf8980-7e8e-11ea-9aa0-311a1a20e5bd.png)

Basically:
 1. Move the text "Ordre d'Achat" to the top right corner.
 1. Move the reference "PO.XXX.123" directly beneath the "Ordre d'Achat" text in the top right corner.

This will also have the effect of giving us more room on the receipt to print text.

**Comment by jniles** [comment]
@kwilu claim
